docs/docs/03-syntax-and-usage/16-rendering-raw-html.md

```diff
@@ -3,7 +3,7 @@
 To render HTML that has come from a trusted source, bypassing all HTML escaping and security mechanisms that templ includes, use the `templ.Raw` function.
 
 :::info
-Only include HTML that comes from a trusted source.
+Only include HTML that comes from a trusted source. Use the `templ.EscapeString` function to escape content that comes from untrusted sources.
```

Comment:
I'm not really sure why you would want to use the `Raw` component, but then escape the content, when you could use a string expression `{ stringContent }` directly, where the content would automatically be HTLM escaped.